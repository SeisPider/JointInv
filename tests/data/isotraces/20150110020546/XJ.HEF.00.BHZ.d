SURF96 R U X -1 94.99999 3.87233 0.23522
SURF96 R U X -1 75.00000 3.77962 0.17692
SURF96 R U X -1 70.00000 3.73125 0.16092
SURF96 R U X -1 65.00000 3.76036 0.15177
SURF96 R U X -1 60.00000 3.83594 0.14578
SURF96 R U X -1 55.00000 3.76816 0.12895
SURF96 R U X -1 50.00000 3.59062 0.10644
SURF96 R U X -1 48.00000 3.58457 0.10184
SURF96 R U X -1 46.00000 3.59202 0.09801
SURF96 R U X -1 44.00000 3.59647 0.09398
SURF96 R U X -1 42.00000 3.59228 0.08950
SURF96 R U X -1 40.00000 3.57986 0.08465
SURF96 R U X -1 38.00000 3.52269 0.07787
SURF96 R U X -1 36.00000 3.43241 0.07003
SURF96 R U X -1 34.00000 3.39720 0.06479
SURF96 R U X -1 32.00000 3.39413 0.06087
SURF96 R U X -1 30.00000 3.35963 0.05591
SURF96 R U X -1 29.00000 3.32700 0.05301
SURF96 R U X -1 28.00000 3.28996 0.05004
SURF96 R U X -1 27.00000 3.25977 0.04738
SURF96 R U X -1 26.00000 3.24834 0.04530
SURF96 R U X -1 25.00000 3.24535 0.04348
SURF96 R U X -1 25.00000 3.47946 0.04998
