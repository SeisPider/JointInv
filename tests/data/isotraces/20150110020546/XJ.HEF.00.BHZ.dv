MFT96 R U T -1 95.256    3.8497    1.0000    6056.0     0.3958E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 93.091    3.8473    1.0000    6056.0     0.4476E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 91.022    3.8447    1.0000    6056.0     0.5038E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 89.043    3.8419    1.0000    6056.0     0.5650E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 87.149    3.8390    1.0000    6056.0     0.6315E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 85.333    3.8359    1.0000    6056.0     0.7038E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 83.592    3.8327    1.0000    6056.0     0.7819E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 81.920    3.8293    1.0000    6056.0     0.8661E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 80.314    3.8258    1.0000    6056.0     0.9562E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 78.769    3.8222    1.0000    6056.0     0.1052E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 77.283    3.8184    1.0000    6056.0     0.1152E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 75.852    3.8146    1.0000    6056.0     0.1257E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 74.473    3.8105    1.0000    6056.0     0.1364E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 73.143    3.8064    1.0000    6056.0     0.1473E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 71.860    3.8022    1.0000    6056.0     0.1583E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 70.621    3.7978    1.0000    6056.0     0.1692E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 69.424    3.7933    1.0000    6056.0     0.1800E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 68.267    3.7887    1.0000    6056.0     0.1906E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 67.148    3.7840    1.0000    6056.0     0.2009E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 66.065    3.7792    1.0000    6056.0     0.2111E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 65.016    3.7743    1.0000    6056.0     0.2212E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 64.000    3.7694    1.0000    6056.0     0.2315E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 63.015    3.7643    1.0000    6056.0     0.2420E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 62.061    3.7591    1.0000    6056.0     0.2532E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 61.134    3.7539    1.0000    6056.0     0.2652E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 60.235    3.7485    1.0000    6056.0     0.2784E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 59.362    3.7431    1.0000    6056.0     0.2929E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 58.514    3.7376    1.0000    6056.0     0.3088E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 57.690    3.7320    1.0000    6056.0     0.3261E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 56.889    3.7264    1.0000    6056.0     0.3449E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 56.110    3.7207    1.0000    6056.0     0.3650E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 55.351    3.7149    1.0000    6056.0     0.3863E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 54.613    3.7091    1.0000    6056.0     0.4086E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 53.895    3.7032    1.0000    6056.0     0.4319E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 53.195    3.6973    1.0000    6056.0     0.4560E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 52.513    3.6913    1.0000    6056.0     0.4812E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 51.848    3.6852    1.0000    6056.0     0.5074E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 51.200    3.6791    1.0000    6056.0     0.5349E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 50.568    3.6730    1.0000    6056.0     0.5639E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 49.951    3.6668    1.0000    6056.0     0.5946E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 49.349    3.6606    1.0000    6056.0     0.6271E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 48.762    3.6543    1.0000    6056.0     0.6614E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 48.188    3.6481    1.0000    6056.0     0.6976E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 47.628    3.6417    1.0000    6056.0     0.7352E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 47.080    3.6354    1.0000    6056.0     0.7740E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 46.545    3.6290    1.0000    6056.0     0.8135E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 46.022    3.6226    1.0000    6056.0     0.8530E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 45.511    3.6162    1.0000    6056.0     0.8920E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 45.011    3.6098    1.0000    6056.0     0.9298E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 44.522    3.6034    1.0000    6056.0     0.9659E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 44.043    3.5969    1.0000    6056.0     0.1000E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 43.574    3.5905    1.0000    6056.0     0.1032E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 43.116    3.5840    1.0000    6056.0     0.1061E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 42.667    3.5775    1.0000    6056.0     0.1087E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 42.227    3.5711    1.0000    6056.0     0.1111E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 41.796    3.5646    1.0000    6056.0     0.1132E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 41.374    3.5581    1.0000    6056.0     0.1151E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 40.960    3.5517    1.0000    6056.0     0.1169E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 40.554    3.5452    1.0000    6056.0     0.1183E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 40.157    3.5387    1.0000    6056.0     0.1196E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 39.767    3.5323    1.0000    6056.0     0.1207E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 39.385    3.5259    1.0000    6056.0     0.1215E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 39.010    3.5195    1.0000    6056.0     0.1221E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 38.642    3.5131    1.0000    6056.0     0.1224E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 38.280    3.5067    1.0000    6056.0     0.1223E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 37.926    3.5003    1.0000    6056.0     0.1219E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 37.578    3.4940    1.0000    6056.0     0.1212E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 37.236    3.4877    1.0000    6056.0     0.1203E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 36.901    3.4814    1.0000    6056.0     0.1191E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 36.571    3.4752    1.0000    6056.0     0.1178E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 36.248    3.4690    1.0000    6056.0     0.1165E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 35.930    3.4628    1.0000    6056.0     0.1153E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 35.617    3.4566    1.0000    6056.0     0.1143E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 35.310    3.4505    1.0000    6056.0     0.1138E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 35.009    3.4444    1.0000    6056.0     0.1137E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 34.712    3.4384    1.0000    6056.0     0.1141E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 34.420    3.4324    1.0000    6056.0     0.1150E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 34.133    3.4264    1.0000    6056.0     0.1164E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 33.851    3.4205    1.0000    6056.0     0.1181E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 33.574    3.4147    1.0000    6056.0     0.1202E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 33.301    3.4089    1.0000    6056.0     0.1224E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 33.032    3.4031    1.0000    6056.0     0.1246E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 32.768    3.3974    1.0000    6056.0     0.1268E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 32.508    3.3917    1.0000    6056.0     0.1290E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 32.252    3.3861    1.0000    6056.0     0.1311E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 32.000    3.3806    1.0000    6056.0     0.1332E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 31.752    3.3751    1.0000    6056.0     0.1354E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 31.508    3.3696    1.0000    6056.0     0.1379E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 31.267    3.3643    1.0000    6056.0     0.1406E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 31.030    3.3589    1.0000    6056.0     0.1438E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 30.797    3.3537    1.0000    6056.0     0.1476E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 30.567    3.3485    1.0000    6056.0     0.1520E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 30.341    3.3434    1.0000    6056.0     0.1571E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 30.118    3.3383    1.0000    6056.0     0.1629E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 29.898    3.3334    1.0000    6056.0     0.1692E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 29.681    3.3284    1.0000    6056.0     0.1761E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 29.468    3.3236    1.0000    6056.0     0.1834E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 29.257    3.3188    1.0000    6056.0     0.1908E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 29.050    3.3141    1.0000    6056.0     0.1984E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 28.845    3.3095    1.0000    6056.0     0.2059E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 28.643    3.3049    1.0000    6056.0     0.2131E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 28.444    3.3005    1.0000    6056.0     0.2200E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 28.248    3.2961    1.0000    6056.0     0.2263E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 28.055    3.2918    1.0000    6056.0     0.2319E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 27.864    3.2875    1.0000    6056.0     0.2367E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 27.676    3.2834    1.0000    6056.0     0.2405E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 27.490    3.2793    1.0000    6056.0     0.2433E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 27.307    3.2753    1.0000    6056.0     0.2448E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 27.126    3.2715    1.0000    6056.0     0.2450E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.947    3.2676    1.0000    6056.0     0.2438E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.771    3.2639    1.0000    6056.0     0.2411E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.597    3.2603    1.0000    6056.0     0.2370E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.426    3.2568    1.0000    6056.0     0.2313E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.256    3.2533    1.0000    6056.0     0.2242E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.089    3.2500    1.0000    6056.0     0.2157E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 25.924    3.2467    1.0000    6056.0     0.2058E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 25.761    3.2436    1.0000    6056.0     0.1948E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 25.600    3.2405    1.0000    6056.0     0.1827E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 25.441    3.2375    1.0000    6056.0     0.1698E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 25.284    3.2347    1.0000    6056.0     0.1562E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 25.129    3.2319    1.0000    6056.0     0.1422E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 95.000    3.8494    1.0000    6056.0     0.4020E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 90.000    3.8432    1.0000    6056.0     0.5354E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 85.000    3.8353    1.0000    6056.0     0.7187E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 80.000    3.8251    1.0000    6056.0     0.9756E-06   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 75.000    3.8121    1.0000    6056.0     0.1323E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 70.000    3.7955    1.0000    6056.0     0.1748E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 65.000    3.7743    1.0000    6056.0     0.2214E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 60.000    3.7471    1.0000    6056.0     0.2823E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 55.000    3.7121    1.0000    6056.0     0.3969E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 50.000    3.6673    1.0000    6056.0     0.5922E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 48.000    3.6459    1.0000    6056.0     0.7102E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 46.000    3.6224    1.0000    6056.0     0.8547E-05   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 44.000    3.5963    1.0000    6056.0     0.1003E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 42.000    3.5677    1.0000    6056.0     0.1122E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 40.000    3.5361    1.0000    6056.0     0.1201E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 38.000    3.5017    1.0000    6056.0     0.1220E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 36.000    3.4641    1.0000    6056.0     0.1155E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 34.000    3.4237    1.0000    6056.0     0.1172E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 32.000    3.3806    1.0000    6056.0     0.1332E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 30.000    3.3357    1.0000    6056.0     0.1663E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 29.000    3.3130    1.0000    6056.0     0.2002E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 28.000    3.2906    1.0000    6056.0     0.2333E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 27.000    3.2688    1.0000    6056.0     0.2441E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.000    3.2482    1.0000    6056.0     0.2103E-04   -5.655000   68.361298   46.770000   85.767998 0  1 COMMENT
