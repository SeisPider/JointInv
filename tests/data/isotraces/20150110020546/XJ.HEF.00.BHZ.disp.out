SURF96 R U T -1 95.000    3.8494    1.0000    
SURF96 R C T -1 95.000    6.3560    1.0000    
SURF96 R U T -1 90.000    3.8432    1.0000    
SURF96 R C T -1 90.000    6.1451    1.0000    
SURF96 R U T -1 85.000    3.8353    1.0000    
SURF96 R C T -1 85.000    5.9466    1.0000    
SURF96 R U T -1 80.000    3.8251    1.0000    
SURF96 R C T -1 80.000    5.7594    1.0000    
SURF96 R U T -1 75.000    3.8121    1.0000    
SURF96 R C T -1 75.000    5.5821    1.0000    
SURF96 R U T -1 70.000    3.7955    1.0000    
SURF96 R C T -1 70.000    5.4134    1.0000    
SURF96 R U T -1 65.000    3.7743    1.0000    
SURF96 R C T -1 65.000    5.2519    1.0000    
SURF96 R U T -1 60.000    3.7471    1.0000    
SURF96 R C T -1 60.000    5.0965    1.0000    
SURF96 R U T -1 55.000    3.7121    1.0000    
SURF96 R C T -1 55.000    4.9455    1.0000    
SURF96 R U T -1 50.000    3.6673    1.0000    
SURF96 R C T -1 50.000    4.7971    1.0000    
SURF96 R U T -1 48.000    3.6459    1.0000    
SURF96 R C T -1 48.000    4.7380    1.0000    
SURF96 R U T -1 46.000    3.6224    1.0000    
SURF96 R C T -1 46.000    4.6788    1.0000    
SURF96 R U T -1 44.000    3.5963    1.0000    
SURF96 R C T -1 44.000    4.6193    1.0000    
SURF96 R U T -1 42.000    3.5677    1.0000    
SURF96 R C T -1 42.000    4.5593    1.0000    
SURF96 R U T -1 40.000    3.5361    1.0000    
SURF96 R C T -1 40.000    4.4985    1.0000    
SURF96 R U T -1 38.000    3.5017    1.0000    
SURF96 R C T -1 38.000    4.4367    1.0000    
SURF96 R U T -1 36.000    3.4641    1.0000    
SURF96 R C T -1 36.000    4.3737    1.0000    
SURF96 R U T -1 34.000    3.4237    1.0000    
SURF96 R C T -1 34.000    4.3090    1.0000    
SURF96 R U T -1 32.000    3.3806    1.0000    
SURF96 R C T -1 32.000    4.2425    1.0000    
SURF96 R U T -1 30.000    3.3357    1.0000    
SURF96 R C T -1 30.000    4.1737    1.0000    
SURF96 R U T -1 29.000    3.3130    1.0000    
SURF96 R C T -1 29.000    4.1385    1.0000    
SURF96 R U T -1 28.000    3.2906    1.0000    
SURF96 R C T -1 28.000    4.1026    1.0000    
SURF96 R U T -1 27.000    3.2688    1.0000    
SURF96 R C T -1 27.000    4.0662    1.0000    
SURF96 R U T -1 26.000    3.2482    1.0000    
SURF96 R C T -1 26.000    4.0292    1.0000    
